Kernel/Syscalls/exit.cpp

```diff
@@ -33,8 +33,10 @@ void Process::sys$exit(int status)
 {
     cli();
 
-    if (status != 0)
+    if (status != 0) {
         dump_backtrace();
+        set_dump_core(true);
```

Comment:
why? I don't think `exit(whatever)` needs a coredump...?

Reply:
Yeah I tend to agree. The current behavior is dumping a backtrace, which is not very useful for userspace without the core dump that `CrashDaemon` symbolicates.
But I don't think we need a backtrace here either, so I went ahead and disabled both printing backtrace & dumping core.